roles/network_plugin/calico/tasks/install.yml

```diff
@@ -158,7 +158,7 @@
         filename: "{{ kube_config_dir }}/kdd-crds.yml"
         state: "latest"
       when:
-        - inventory_hostname == groups['kube_control_plane'][0]
+        - inventory_hostname == first_kube_control_plane
```

Comment:
It is not appropriate to apply these yaml files on every node here, because there may be conflicts and errors when applying them concurrently, and it also wastes time.